apps/test/unit/templates/sectionsRefresh/CoteacherSettingsTest.jsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import {mount, shallow} from 'enzyme';
+import {expect} from '../../../util/reconfiguredChai';
+import CoteacherSettings from '@cdo/apps/templates/sectionsRefresh/CoteacherSettings';
+import sinon from 'sinon';
+
+const testSectionInstructors = [
+  {instructorName: 'Allosaurus', instructorEmail: 'allosaurus@code.org'},
+  {instructorName: 'Brachiosaurus', instructorEmail: 'brachiosaurus@code.org'},
+];
+
+describe('CoteacherSettings', () => {
+  it('renders count of existing coteachers', () => {
+    const wrapper = mount(
+      <CoteacherSettings
+        sectionInstructors={testSectionInstructors}
+        addCoteacher={() => {}}
+        coteachersToAdd={[]}
+      />
+    );
+    expect(wrapper.find('Figcaption')).to.have.lengthOf(1);
+    expect(wrapper.find('Figcaption').text()).to.include(
```

Comment:
I recently started using `expect(wrapper.find('Figcaption').props().children).to.equal(
      '1/5 co-teachers added'
    );` in something like this so I could use `shallow` instead of `mount` for a test like this.  I think it would work in this case too.